internal/parser/grammar/IMAPParser.g4

```diff
@@ -36,7 +40,9 @@ quotedChar
 
 quotedSpecial: DQuote | Backslash;
 
-literal: LCurly number RCurly crlf uuid;
+literal locals[
+std::string mLiteralText
+]: LCurly number{auto numberCtx=$number.ctx;}RCurly crlf {$mLiteralText = $parser->consumeTokens(numberCtx->getText());};
```

Comment:
mmm okay i was wondering how you did this, and i see you used injected code -- i guess at this point we've accepted being tied to C++, i think this is a good choice :+1: